README.md

```diff
@@ -167,6 +167,33 @@ app.sessionEnded(function(request, response) {
 });
 ```
 
+## AudioPlayer Event Request
+
+Define the handler for multiple events using multiple calls to `audioPlayer()`. You can define only one handler per event. Event handlers that don't return an immediate response (because they do some asynchronous operation) must return false. 
```

Comment:
Maybe add a link to audio player support on some amazon site to explain what this is somewhere?